feedthefox/users/adapters.py

```diff
@@ -59,6 +60,7 @@ def populate_user(self, request, sociallogin, data):
 
         if mozillian:
             user.username = mozillian['username']
```

Comment:
yes because we need something to populate the instance. If we have a duplicate, the custom save will make it unique.